webpack.common.js

```diff
@@ -1,23 +1,23 @@
-const path = require('path');
+const path = require("path");
 
 module.exports = {
-    entry: './src/index.ts',
+    entry: "./src/index.ts",
     module: {
         rules: [
             {
                 test: /\.tsx?$/,
-                use: 'ts-loader',
-                exclude: /node_modules/,
-            },
-        ],
+                use: "ts-loader",
+                exclude: /node_modules/
+            }
+        ]
     },
     resolve: {
-        extensions: ['.ts', '.js']
+        extensions: [".ts", ".js"]
     },
     output: {
-        filename: 'main.js',
-        path: path.resolve(__dirname, 'dist'),
-        library: 'openInsights',
-        libraryTarget: 'window',
-    },
+        filename: "main.js",
+        path: path.resolve(__dirname, "dist"),
+        library: "openInsights",
```

Comment:
Maybe add a //TODO comment here that this may need to change depending on what we decide to call this project.